### DISCUSSION TITLE
Cilium eBPF using Netronome SmartNIC and throughput test doesn't work

### DISCUSSION THREAD

**Discussion by hyeongwooo** [cilium/ebpf#1274] [Q&A]
HI i have a question!
I have installed a netronome SmartNIC supporting eBPF acceleration on two servers and connected them via a separate L2 switch using a 40G optical cable. Additionally, I've installed Kubernetes and Cilium version 1.14.5, along with the NIC drivers and firmware. When conducting throughput tests using netperf, I noticed that when sending data from the master node to the worker node, I achieved speeds close to the line rate. However, when sending data from a pod on the master node to a pod on the worker node, the speed decreased by 25%. Without installing the firmware and drivers, I achieve line rate speeds. Could this indicate an issue with the drivers and firmware?

**Comment by lmb** [comment]
Hi, this repository is used for the ebpf-go library. It seems like you having more general questions which we can't help you with, sorry. Please try the [Cilium slack](https://cilium.herokuapp.com/).
